### PR TITLE
Issue 6 mark all read functionality as props

### PR DESCRIPTION
**Pull request by NagarjunShroff**
Issue : [https://github.com/atapas/notifyme/issues/6](url)

Now users can control the functionality of the "Mark All As Read" button bypassing their own function as a prop.

### CODE REVIEW

File: NotifyMe.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ const NotifyMe = props => {
                         <Popover.Title as="h3">{heading}</Popover.Title>
                         <Popover.Content style={{ padding: '3px 3px' }}>
                             {showCount && <div>
-                                <Button variant="link" onClick={markAsRead}>
+                                <Button variant="link" onClick={props.markAsRead || markAsRead}>
```
Comment (atapas) [inline NotifyMe.js:167]: Can we rename the prop as `markAsReadFn` so that it sounds like a function?

File: NotifyMe.js [hunk 2]
```diff
@@ -164,7 +164,7 @@ const NotifyMe = props => {
...
                                     <BookOpen size={24} />
                                     Mark all as read
```
Comment (atapas) [inline NotifyMe.js:169]: The bottom of the file has the `props` validations. It should be there as an optional prop and function as type.

File: README.md [hunk 3]
```diff
@@ -171,6 +172,18 @@ Here is an example usage,
     <td> No </td>
     <td> Any string of your choice as a key. Default value is, <b>notification_timeline_storage_id</b></td>
   </tr>
+  
+  <tr>
+    <td> markAsRead </td>
```
Comment (atapas) [inline README.md:177]: Renaming it accordingly

File: README.md [hunk 4]
```diff
@@ -171,6 +172,18 @@ Here is an example usage,
...
+    <td> User can control the functionality of "Mark All As Read" by passing the function as prop as below
+    markAsRead = {() => yourOwnFunction()}
+    </td>
+    <td> No </td>
+    <td> Now "Mark All As Read" can be controlled by passing you own function as prop. Default functionality is,
```
Comment (atapas) [inline README.md:182]: Remove the word `Now`.

Comment (atapas) [inline README.md:182]: `passing you own function` => Typo.

**Review by atapas — CHANGES_REQUESTED** [review]
Please have a look at the review comments.

**Comment by atapas** [comment]
@all-contributors please add @NagarjunShroff  for Code

File: NotifyMe.js (hunk 1 above)
Comment (NagarjunShroff) [reply NotifyMe.js:167]: sure

File: README.md (hunk 4 above)
Comment (NagarjunShroff) [reply README.md]: done

Comment (NagarjunShroff) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (NagarjunShroff) [reply README.md:177]: done

File: NotifyMe.js (hunk 2 above)
Comment (NagarjunShroff) [reply NotifyMe.js:169]: done

**Review by atapas — APPROVED** [review]
Changes are looking good for merge

**Pull request merged**